bundle/src/main/java/com/adobe/acs/commons/redirectmaps/models/package-info.java

```diff
@@ -17,5 +17,5 @@
  * limitations under the License.
  * #L%
  */
-@aQute.bnd.annotation.Version("3.14.4")
+@aQute.bnd.annotation.Version("4.0.0")
```

Comment:
Maybe we need to add a comment here that this package is only meant for use by ACS AEM Commons internals. We really need an `@VisibleForScripting` annotation like Guava has `@VisibleForTesting`. But we have a few places where this is true, so no need to necessarily deal with it right now.